client/asset/dcr/rpcwallet.go

```diff
@@ -64,6 +68,8 @@ type rpcWallet struct {
 
 // Ensure rpcWallet satisfies the Wallet interface.
 var _ Wallet = (*rpcWallet)(nil)
+var _ Mempooler = (*rpcWallet)(nil)
+var _ FeeRateEstimator = (*rpcWallet)(nil)
```

Comment:
I get where you're going with this, where a native/integrated dcrwallet would likely be known to be exclusively SPV, but the `rpcWallet` can be either full-node backed or SPV.  On top of that, we don't find out if it's SPV mode until it connects.

Reply:
And on reconnection as well. I've toyed with shutting down dcrwallet after dexc initially connected to it and restarted dcrwallet with spv where it was previously using full node. The exchange wallet notices when this change occurs on reconnect.

Reply:
As a partial resolution, at the time of assertion we could add the SpvMode condition:

```go
mempooler, is := dcr.wallet.(Mempooler)
if !is || dcr.wallet.SpvMode() {
	return